lrpar/cttests/src/lib.rs

```diff
@@ -388,20 +394,32 @@ fn test_grmtools_section() {
         "%grmtools{a, x: y(z)}",
         "%grmtools{a, !b, x: y(z), e: f}",
         "%grmtools{a, !b, x: y(z), e: f,}",
+        "%grmtools{a, !b, x: w::y(z), e: f}",
+        "%grmtools{a, !b, x: w::y(z), e: f,}",
+        "%grmtools{a, !b, x: w::y(z), e: g::f}",
+        "%grmtools{a, !b, x: w::y(z), e: g::f,}",
     ];
 
     let lexerdef = grmtools_section_l::lexerdef();
     for src in srcs {
         let l = lexerdef.lexer(src);
-        let (val, errs) = grmtools_section_y::parse(&l);
+        let (yacc_parsed, errs) = grmtools_section_y::parse(&l);
+        let mut parse_error = false;
+
         if !errs.is_empty() {
             let mut s = "Errors:\n".to_string();
-            for e in errs {
+            for e in &errs {
                 s.push_str(&format!("\t{}\n", e));
             }
-            panic!("{}", s);
+            eprintln!("{}", s);
+            parse_error = true;
         }
-        println!("{:?}", val);
+        let parser = cfgrammar::header::GrmtoolsSectionParser::new(src, true);
+        let res = parser.parse();
+        let (header_parsed, _) = res.unwrap();
+        assert_eq!(yacc_parsed, Some(header_parsed));
+        assert!(!parse_error);
+        println!("{:?} {:?}", src, yacc_parsed);
```

Comment:
I think we should kill the debugging `print`s/`eprint`s in tests.

Reply:
Yeah, It is mostly for the moment, current I *often* find myself running `cargo test -- --nocapture | grep '%grmtools'`,


But I'll remove them once this is up to form, I've gone ahead and marked as draft since it seems I still have a bit of work to go before I would be comfortable considering replacing the other parsers with it.

Reply:
I *think* i got all of them removed in [79fb45b](https://github.com/softdevteam/grmtools/pull/544/commits/79fb45b811682e274553cad36b6993069944e50f)